cookbook/logging/monolog.rst

```diff
@@ -236,6 +236,52 @@ only for a specific handler.
     generate hundreds of log lines. Consider using tools like the `logrotate`_
     Linux command to rotate log files before they become a problem.
 
+    In case you cannot use a dedicated tool for rotating log files, consider using
```

Comment:
@javiereguiluz Thanks for adding this :). I think we should move this to a whole new section - with a header like `How to Rotate your Log Files` so it can be easily found. OR, make a new, very-short cookbook entry (to be even easier to find).

Thanks!